Upstream bug: http://bugs.mysql.com/bug.php?id=69342

# Amalgamation of patches to various mysql man pages

# ===== mysqladmin manual page =====

diff -up mysql-5.6.15/man/mysql.1.p24 mysql-5.6.15/man/mysql.1
--- mysql-5.6.15/man/mysql.1.p24	2013-11-17 19:03:04.000000000 +0100
+++ mysql-5.6.15/man/mysql.1	2013-12-10 12:28:29.055463911 +0100
@@ -135,7 +135,8 @@ Section\ \&4.2.3.4, \(lqCommand-Line Opt
 .IP \(bu 2.3
 .\}
 .\" mysql: help option.\" help option: mysql\fB\-\-help\fR,
-\fB\-?\fR
+\fB\-?\fR,
+\fB\-I\fR
 .sp
 Display a help message and exit\&.
 .RE
@@ -346,6 +347,21 @@ Section\ \&6.3.6, \(lqPassword Expiratio
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysql: connect-timeout option
+.\" connect-timeout option: mysql
+\fB\-\-connect\-timeout=\fR\fB\fIseconds\fR\fR
+.sp
+Set the time for connection timeout in seconds\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysql: database option.\" database option: mysql\fB\-\-database=\fR\fB\fIdb_name\fR\fR,
 \fB\-D \fR\fB\fIdb_name\fR\fR
 .sp
@@ -445,6 +461,37 @@ Section\ \&10.5, \(lqCharacter Set Confi
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysql: defaults-file option
+.\" defaults-file option: mysql
+\fB\-\-defaults-file=\fR\fB\fIfilename\fR\fR
+.sp
+Set \fB\fIfilename\fR\fR as the file to read default options from, override global defaults files\&. Must be given as first option\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysql: defaults-extra-file option
+.\" defaults-extra-file option: mysql
+\fB\-\-defaults-extra-file=\fR\fB\fIfilename\fR\fR
+.sp
+Set \fB\fIfilename\fR\fR as the file to read default options from after the global defaults files has been read\&.
+Must be given as first option\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysql: delimiter option.\" delimiter option: mysql\fB\-\-delimiter=\fR\fB\fIstr\fR\fR
 .sp
 Set the statement delimiter\&. The default is the semicolon character (\(lq;\(rq)\&.
@@ -636,6 +683,36 @@ has no effect if the server does not als
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysql: max-allowed-packet option
+.\" max-allowed-packet option: mysql
+\fB\-\-max\-allowed\-packet=\fR\fB\fInum\fR\fR
+.sp
+Specify the maximum packet length to send to or recieve form server\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysql: max-join-size option
+.\" max-join-size option: mysql
+\fB\-\-max\-join\-size=\fR\fB\fInum\fR\fR
+.sp
+Set the automatic limit for rows in a join when using \fB\-\-safe\-updates\fR\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysql: named-commands option.\" named-commands option: mysql\fB\-\-named\-commands\fR,
 \fB\-G\fR
 .sp
@@ -659,11 +736,26 @@ the section called \(lqMYSQL COMMANDS\(r
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysql: net-buffer-length option
+.\" net-buffer-length option: mysql
+\fB\-\-net\-buffer\-lenght=\fR\fB\fIsize\fR\fR
+.sp
+Set the buffer size for TCP/IP and socket communication\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysql: no-auto-rehash option.\" no-auto-rehash option: mysql\fB\-\-no\-auto\-rehash\fR,
 \fB\-A\fR
 .sp
 This has the same effect as
-\fB\-skip\-auto\-rehash\fR\&. See the description for
+\fB\-\-skip\-auto\-rehash\fR\&. See the description for
 \fB\-\-auto\-rehash\fR\&.
 .RE
 .sp
@@ -689,6 +781,21 @@ Do not beep when errors occur\&.
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysql: no-defaults option
+.\" no-defaults option: mysql
+\fB\-\-no\-defaults\fR
+.sp
+Do not read default options from any option file\&. This must be given as the first argument\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysql: one-database option.\" one-database option: mysql\fB\-\-one\-database\fR,
 \fB\-o\fR
 .sp
@@ -884,6 +991,21 @@ The TCP/IP port number to use for the co
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysql: print-defaults option
+.\" print-defaults option: mysql
+\fB\-\-print\-defaults\fR
+.sp
+Print the program argument list and exit\&. This must be given as the first argument\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysql: prompt option.\" prompt option: mysql\fB\-\-prompt=\fR\fB\fIformat_str\fR\fR
 .sp
 Set the prompt to the specified format\&. The default is
@@ -1075,6 +1197,36 @@ and renamed in 5\&.6\&.7 to
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysql: select-limit option
+.\" select-limit option: mysql
+\fB\-\-select\-limit=\fR\fB\fIlimit\fR\fR
+.sp
+Set automatic limit for SELECT when using \fB\-\-safe\-updates\fR\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysql: server-arg option
+.\" server-arg option: mysql
+\fB\-\-server\-arg=\fR\fB\fIname\fR\fR
+.sp
+Send \fB\fIname\fR\fR as a parameter to the embedded server\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysql: show-warnings option.\" show-warnings option: mysql\fB\-\-show\-warnings\fR
 .sp
 Cause warnings to be shown after each statement if there are any\&. This option applies to interactive and batch mode\&.
@@ -1122,6 +1274,21 @@ option\&.
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysql: skip-auto-rehash option
+.\" skip-auto-rehash option: mysql
+\fB\-\-skip\-auto\-rehash\fR
+.sp
+Disable automatic rehashing\&. Synonym for \fB\-\-disable\-auto\-rehash\fR\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysql: skip-column-names option.\" skip-column-names option: mysql\fB\-\-skip\-column\-names\fR,
 \fB\-N\fR
 .sp
@@ -1168,7 +1335,7 @@ localhost, the Unix socket file to use,
 .\" mysql: SSL options.\" SSL options: mysql\fB\-\-ssl*\fR
 .sp
 Options that begin with
-\fB\-\-ssl\fR
+\fB\-\-ssl\fR,
 specify whether to connect to the server using SSL and indicate where to find SSL keys and certificates\&. See
 Section\ \&6.3.9.4, \(lqSSL Command Options\(rq\&.
 .RE
@@ -1326,7 +1493,7 @@ The XML output also uses an XML namespac
 .RS 4
 .\}
 .nf
-shell> \fBmysql \-\-xml \-uroot \-e "SHOW VARIABLES LIKE \*(Aqversion%\*(Aq"\fR
+shell> \fBmysql \-\-xml \-u root \-e "SHOW VARIABLES LIKE \*(Aqversion%\*(Aq"\fR
 <?xml version="1\&.0"?>
 <resultset statement="SHOW VARIABLES LIKE \*(Aqversion%\*(Aq" xmlns:xsi="http://www\&.w3\&.org/2001/XMLSchema\-instance">
 <row>
@@ -3188,16 +3355,16 @@ statements that probably need to examine
 .RE
 .PP
 To specify limits different from 1,000 and 1,000,000, you can override the defaults by using the
-\fB\-\-select_limit\fR
+\fB\-\-select\-limit\fR
 and
-\fB\-\-max_join_size\fR
+\fB\-\-max\-join\-size\fR
 options:
 .sp
 .if n \{\
 .RS 4
 .\}
 .nf
-shell> \fBmysql \-\-safe\-updates \-\-select_limit=500 \-\-max_join_size=10000\fR
+shell> \fBmysql \-\-safe\-updates \-\-select\-limit=500 \-\-max\-join\-size=10000\fR
 .fi
 .if n \{\
 .RE
diff -up mysql-5.6.15/man/mysqladmin.1.p24 mysql-5.6.15/man/mysqladmin.1
--- mysql-5.6.15/man/mysqladmin.1.p24	2013-11-17 19:03:04.000000000 +0100
+++ mysql-5.6.15/man/mysqladmin.1	2013-12-10 13:05:23.105401084 +0100
@@ -639,6 +639,21 @@ Compress all information sent between th
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysqladmin: connect-timeout option
+.\" connect-timeout option: mysqladmin
+\fB\-\-connect-timeout=\fR\fB\fItimeout\fR\fR
+.sp
+Equivalent to \fB\-\-connect_timeout\fR, see the end of this section\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysqladmin: count option.\" count option: mysqladmin\fB\-\-count=\fR\fB\fIN\fR\fR,
 \fB\-c \fR\fB\fIN\fR\fR
 .sp
@@ -729,6 +744,37 @@ Section\ \&10.5, \(lqCharacter Set Confi
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysqladmin: defaults-extra-file option
+.\" defaults-extra-file option: mysqladmin
+\fB\-\-defaults\-extra\-file=\fR\fB\fIfilename\fR\fR
+.sp
+Set \fB\fIfilename\fR\fR as the file to read default options from after the global defaults files has been read\&.
+Must be given as first option\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqladmin: defaults-file option
+.\" defaults-file option: mysqladmin
+\fB\-\-defaults\-file=\fR\fB\fIfilename\fR\fR
+.sp
+Set \fB\fIfilename\fR\fR as the file to read default options from, override global defaults files\&. Must be given as first option\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysqladmin: enable-cleartext-plugin option.\" enable-cleartext-plugin option: mysqladmin\fB\-\-enable\-cleartext\-plugin\fR
 .sp
 Enable the
@@ -789,6 +835,21 @@ Suppress the warning beep that is emitte
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysqladmin: no-defaults option
+.\" no-defaults option: mysqladmin
+\fB\-\-no\-defaults\fR
+.sp
+Do not read default options from any option file\&. This must be given as the first argument\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysqladmin: password option.\" password option: mysqladmin\fB\-\-password[=\fR\fB\fIpassword\fR\fR\fB]\fR,
 \fB\-p[\fR\fB\fIpassword\fR\fR\fB]\fR
 .sp
@@ -862,6 +923,21 @@ The TCP/IP port number to use for the co
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysqladmin: print-defaults option
+.\" print-defaults option: mysqladmin
+\fB\-\-print\-defaults\fR
+.sp
+Print the program argument list and exit\&. This must be given as the first argument\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysqladmin: protocol option.\" protocol option: mysqladmin\fB\-\-protocol={TCP|SOCKET|PIPE|MEMORY}\fR
 .sp
 The connection protocol to use for connecting to the server\&. It is useful when the other connection parameters normally would cause a protocol to be used other than the one you want\&. For details on the permissible values, see
@@ -938,6 +1014,21 @@ localhost, the Unix socket file to use,
 .RE
 .sp
 .RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqladmin: shutdown-timeout option
+.\" shutdown-timeout option: mysqladmin
+\fB\-\-shutdown\-timeout\fR\fB\fItimeout\fR\fR
+.sp
+Equivalent of \fB\-\-shutdown_timeout\fR, see the end of this section\&.
+.RE
+.sp
+.RS 4
 .ie n \{\
 \h'-04'\(bu\h'+03'\c
 .\}
diff -up mysql-5.6.15/man/mysqlbinlog.1.p24 mysql-5.6.15/man/mysqlbinlog.1
--- mysql-5.6.15/man/mysqlbinlog.1.p24	2013-11-17 19:03:04.000000000 +0100
+++ mysql-5.6.15/man/mysqlbinlog.1	2013-12-09 19:08:52.750074832 +0100
@@ -602,6 +602,7 @@ Do not display any of the groups listed
 \fB\-F\fR
 .sp
 Read binary log files even if they are open or were not closed properly\&.
+Enabled by default, use \fB\-\-skip\-force\-if\-open\fR to disable\&.
 .RE
 .sp
 .RS 4
@@ -715,6 +716,22 @@ entries in the log\&.
 .RE
 .sp
 .RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqlbinlog: open-files-limit option
+.\" open-files-limit option: mysqlbinlog
+\fB\-\-open\-files\-limit=\fR\fB\fINUM\fR\fR
+.sp
+Sets the open_files_limit variable, which is used to reserve file descriptors for
+\fBmysqlbinlog\fR\&.
+.RE
+.sp
+.RS 4
 .ie n \{\
 \h'-04'\(bu\h'+03'\c
 .\}
diff -up mysql-5.6.15/man/mysqlcheck.1.p24 mysql-5.6.15/man/mysqlcheck.1
--- mysql-5.6.15/man/mysqlcheck.1.p24	2013-11-17 19:03:04.000000000 +0100
+++ mysql-5.6.15/man/mysqlcheck.1	2013-12-10 13:02:43.841276619 +0100
@@ -456,6 +456,38 @@ Section\ \&10.5, \(lqCharacter Set Confi
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysqlcheck: defaults-extra-file option
+.\" defaults-extra-file option: mysqlcheck
+\fB\-\-defaults\-extra\-file=\fR\fB\fIfilename\fR\fR
+.sp
+Set \fB\fIfilename\fR\fR as the file to read default options from after the global defaults files has been read\&.
+Must be given as first option\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqlcheck: defaults-file option
+.\" defaults-file option: mysqlcheck
+\fB\-\-defaults\-file=\fR\fB\fIfilename\fR\fR
+.sp
+Set \fB\fIfilename\fR\fR as the file to read default options from, override global defaults files\&.
+Must be given as first option\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysqlcheck: extended option.\" extended option: mysqlcheck\fB\-\-extended\fR,
 \fB\-e\fR
 .sp
@@ -572,6 +604,21 @@ operation\&. This finds only 99\&.99% of
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysqlcheck: no-defaults option
+.\" no-defaults option: mysqlcheck
+\fB\-\-no\-defaults\fR
+.sp
+Do not read default options from any option file\&. This must be given as the first argument\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysqlcheck: optimize option.\" optimize option: mysqlcheck\fB\-\-optimize\fR,
 \fB\-o\fR
 .sp
@@ -712,6 +759,22 @@ Silent mode\&. Print only error messages
 .RE
 .sp
 .RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqlcheck: print-defaults option
+.\" print-defaults option: mysqlcheck
+\fB\-\-print\-defaults\fR
+.sp
+Print the program argument list and exit\&.
+This must be given as the first argument\&.
+.RE
+.sp
+.RS 4
 .ie n \{\
 \h'-04'\(bu\h'+03'\c
 .\}
diff -up mysql-5.6.15/man/mysql_config.1.p24 mysql-5.6.15/man/mysql_config.1
--- mysql-5.6.15/man/mysql_config.1.p24	2013-12-09 19:08:52.751074834 +0100
+++ mysql-5.6.15/man/mysql_config.1	2013-12-10 13:09:46.944577176 +0100
@@ -174,6 +174,22 @@ The default Unix socket file, defined wh
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysql_config: variable option
+.\" variable option: mysql_config
+\fB\-\-variable=VAR\fR
+.sp
+Path to MySQL include, library and plugin directories\&. \fBVAR\fR is one of
+`pkgincludedir`, `pkglibdir` and `plugindir`, respectively\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysql_config: version option.\" version option: mysql_config\fB\-\-version\fR
 .sp
 Version number for the MySQL distribution\&.
diff -up mysql-5.6.15/man/mysqldump.1.p24 mysql-5.6.15/man/mysqldump.1
--- mysql-5.6.15/man/mysqldump.1.p24	2013-11-17 19:03:04.000000000 +0100
+++ mysql-5.6.15/man/mysqldump.1	2013-12-10 13:00:43.204169470 +0100
@@ -1154,7 +1154,8 @@ statements that include column names\&.
 .sp -1
 .IP \(bu 2.3
 .\}
-.\" mysqldump: create-options option.\" create-options option: mysqldump\fB\-\-create\-options\fR
+.\" mysqldump: create-options option.\" create-options option: mysqldump\fB\-\-create\-options\fR,
+\fB\-a\fR
 .sp
 Include all MySQL\-specific table options in the
 CREATE TABLE
@@ -1666,6 +1667,38 @@ transactional options, primarily for the
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysqldump: defaults-extra-file option
+.\" defaults-extra-file option: mysqldump
+\fB\-\-defaults\-extra\-file=\fR\fB\fIfilename\fR\fR
+.sp
+Set \fB\fIfilename\fR\fR as the file to read default options from after the global defaults files has been read\&.
+Must be given as first option\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqldump: defaults-file option
+.\" defaults-file option: mysqldump
+\fB\-\-defaults\-file=\fR\fB\fIfilename\fR\fR
+.sp
+Set \fB\fIfilename\fR\fR as the file to read default options from, override global defaults files\&.
+Must be given as first option\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysqldump: delayed-insert option.\" delayed-insert option: mysqldump\fB\-\-delayed\-insert\fR
 .sp
 For those nontransactional tables that support the
@@ -1917,6 +1950,36 @@ at the end of the option list\&.
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysqldump: max-allowed-packet option
+.\" max-allowed-packet option: mysqldump
+\fB\-\-max\-allowed\-packet=\fR\fB\fIlength\fR\fR
+.sp
+Sets the maximum packet length to send to or recieve from server\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqldump: net-buffer-length option
+.\" net-buffer-length option: mysqldump
+\fB\-\-net\-buffer\-length=\fR\fB\fIlength\fR\fR
+.sp
+Sets the buffer size for TCP/IP and socket communication\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysqldump: no-autocommit option.\" no-autocommit option: mysqldump\fB\-\-no\-autocommit\fR
 .sp
 Enclose the
@@ -2012,6 +2075,21 @@ is on by default\&. Thus you rarely if e
 .el \{\
 .sp -1
 .IP \(bu 2.3
+.\" mysqldump: no-defaults option
+.\" no-defaults option: mysqldump
+\fB\-\-no\-defaults\fR
+.sp
+Do not read default options from any option file\&. This must be given as the first argument\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\}
 The
 \fB\-\-compact\fR
@@ -2184,6 +2262,21 @@ is on by default\&.)
 .RE
 .sp
 .RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqldump: print-defaults option
+.\" print-defaults option: mysqldump
+\fB\-\-print\-defaults\fR
+.sp
+Print the program argument list and exit\&. This must begiven as the first argument\&.
+.RE
+.sp
+.RS 4
 .ie n \{\
 \h'-04'\(bu\h'+03'\c
 .\}
diff -up mysql-5.6.15/man/mysqlimport.1.p24 mysql-5.6.15/man/mysqlimport.1
--- mysql-5.6.15/man/mysqlimport.1.p24	2013-11-17 19:03:04.000000000 +0100
+++ mysql-5.6.15/man/mysqlimport.1	2013-12-10 12:55:58.039851937 +0100
@@ -238,8 +238,40 @@ This option was added in MySQL 5\&.6\&.2
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysqlimport: defaults-extra-file option
+.\" defaults-extra-file option: mysqlimport
+\fB\-\-defaults\-extra\-file=\fR\fB\fIfilename\fR\fR
+.sp
+Set \fB\fIfilename\fR\fR as the file to read default options from after the global defaults files has been
+read\&.  Must be given as first option\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqlimport: defaults-file option
+.\" defaults-file option: mysqlimport
+\fB\-\-defaults\-file=\fR\fB\fIfilename\fR\fR
+.sp
+Set \fB\fIfilename\fR\fR as the file to read default options from, override global defaults files\&.
+Must be given as first option\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysqlimport: delete option.\" delete option: mysqlimport\fB\-\-delete\fR,
-\fB\-D\fR
+\fB\-d\fR
 .sp
 Empty the table before importing the text file\&.
 .RE
@@ -397,6 +429,22 @@ MERGE)\&.
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysqlimport: no-defaults option
+.\" no-defaults option: mysqlimport
+\fB\-\-no\-defaults\fR
+.sp
+Do not read default options from any option file\&. This must be given as the
+first argument\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysqlimport: password option.\" password option: mysqlimport\fB\-\-password[=\fR\fB\fIpassword\fR\fR\fB]\fR,
 \fB\-p[\fR\fB\fIpassword\fR\fR\fB]\fR
 .sp
@@ -513,6 +561,22 @@ Silent mode\&. Produce output only when
 .RE
 .sp
 .RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqlimport: print-defaults option
+.\" print-defaults option: mysqlimport
+\fB\-\-print\-defaults\fR
+.sp
+Print the program argument list and exit\&.
+This must be given as the first argument\&.
+.RE
+.sp
+.RS 4
 .ie n \{\
 \h'-04'\(bu\h'+03'\c
 .\}
diff -up mysql-5.6.15/man/mysqlshow.1.p24 mysql-5.6.15/man/mysqlshow.1
--- mysql-5.6.15/man/mysqlshow.1.p24	2013-11-17 19:03:04.000000000 +0100
+++ mysql-5.6.15/man/mysqlshow.1	2013-12-10 13:08:16.657520289 +0100
@@ -167,7 +167,8 @@ This option is supported beginning with
 .sp -1
 .IP \(bu 2.3
 .\}
-.\" mysqlshow: character-sets-dir option.\" character-sets-dir option: mysqlshow\fB\-\-character\-sets\-dir=\fR\fB\fIpath\fR\fR
+.\" mysqlshow: character-sets-dir option.\" character-sets-dir option: mysqlshow\fB\-\-character\-sets\-dir=\fR\fB\fIpath\fR\fR,
+\fB\-c\fR
 .sp
 The directory where character sets are installed\&. See
 Section\ \&10.5, \(lqCharacter Set Configuration\(rq\&.
@@ -285,6 +286,38 @@ This option was added in MySQL 5\&.6\&.2
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysqlshow: defaults-extra-file option
+.\" defaults-extra-file option: mysqlshow
+\fB\-\-defaults\-extra\-file=\fR\fB\fIfilename\fR\fR
+.sp
+Set \fB\fIfilename\fR\fR as the file to read default options from after the global defaults files has been
+read\&.  Must be given as first option\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqlshow: defaults-file option
+.\" defaults-file option: mysqlshow
+\fB\-\-defaults\-file=\fR\fB\fIfilename\fR\fR
+.sp
+Set \fB\fIfilename\fR\fR as the file to read default options from, override global defaults files\&.
+Must be given as first option\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysqlshow: host option.\" host option: mysqlshow\fB\-\-host=\fR\fB\fIhost_name\fR\fR,
 \fB\-h \fR\fB\fIhost_name\fR\fR
 .sp
@@ -313,6 +346,22 @@ Show table indexes\&.
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysqlshow: no-defaults option
+.\" no-defaults option: mysqlshow
+\fB\-\-no\-defaults\fR
+.sp
+Do not read default options from any option file\&. This must be given as the
+first argument\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysqlshow: password option.\" password option: mysqlshow\fB\-\-password[=\fR\fB\fIpassword\fR\fR\fB]\fR,
 \fB\-p[\fR\fB\fIpassword\fR\fR\fB]\fR
 .sp
@@ -413,6 +462,22 @@ VIEW\&.
 .RE
 .sp
 .RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqlshow: print-defaults option
+.\" print-defaults option: mysqlshow
+\fB\-\-print\-defaults\fR
+.sp
+Print the program argument list and exit\&.
+This must be given as the first argument\&.
+.RE
+.sp
+.RS 4
 .ie n \{\
 \h'-04'\(bu\h'+03'\c
 .\}
diff -up mysql-5.6.15/man/mysqlslap.1.p24 mysql-5.6.15/man/mysqlslap.1
--- mysql-5.6.15/man/mysqlslap.1.p24	2013-11-17 19:03:04.000000000 +0100
+++ mysql-5.6.15/man/mysqlslap.1	2013-12-10 13:11:04.721623937 +0100
@@ -531,6 +531,38 @@ Section\ \&6.3.7.7, \(lqThe Cleartext Cl
 .el \{\
 .sp -1
 .IP \(bu 2.3
+.\" mysqlslap: defaults-extra-file option
+.\" defaults-extra-file option: mysqlslap
+\fB\-\-defaults\-extra\-file=\fR\fB\fIfilename\fR\fR
+.sp
+Set \fB\fIfilename\fR\fR as the file to read default options from after the global defaults files has been
+read\&.  Must be given as first option\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqlslap: defaults-file option
+.\" defaults-file option: mysqlslap
+\fB\-\-defaults\-file=\fR\fB\fIfilename\fR\fR
+.sp
+Set \fB\fIfilename\fR\fR as the file to read default options from, override global defaults files\&.
+Must be given as first option\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\}
 .\" mysqlslap: engine option.\" engine option: mysqlslap\fB\-\-engine=\fR\fB\fIengine_name\fR\fR,
 \fB\-e \fR\fB\fIengine_name\fR\fR
@@ -589,6 +621,37 @@ from dropping any schema it creates duri
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysqlslap: no-defaults option
+.\" no-defaults option: mysqlslap
+\fB\-\-no\-defaults\fR
+.sp
+Do not read default options from any option file\&. This must be given as the
+first argument\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqlslap: no-drop option
+.\" no-drop option: mysqlslap
+\fB\-\-no\-drop\fR
+.sp
+Do not drop the schema after the test\&.
+.RE
+.sp
+.RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
 .\" mysqlslap: number-char-cols option.\" number-char-cols option: mysqlslap\fB\-\-number\-char\-cols=\fR\fB\fIN\fR\fR,
 \fB\-x \fR\fB\fIN\fR\fR
 .sp
@@ -819,6 +882,22 @@ Section\ \&4.2.2, \(lqConnecting to the
 .RE
 .sp
 .RS 4
+.ie n \{\
+\h'-04'\(bu\h'+03'\c
+.\}
+.el \{\
+.sp -1
+.IP \(bu 2.3
+.\}
+.\" mysqlslap: print-defaults option
+.\" print-defaults option: mysqlslap
+\fB\-\-print\-defaults\fR
+.sp
+Print the program argument list and exit\&.
+This must be given as the first argument\&.
+.RE
+.sp
+.RS 4
 .ie n \{\
 \h'-04'\(bu\h'+03'\c
 .\}
